app/templates/press-releases/press-release/shared/edit.hbs

```diff
@@ -1,2 +1,83 @@
 {{page-title "Bewerk"}}
+<Auk::Navbar @skin="gray-100" @size="large">
+  <Auk::Toolbar>
+    <Auk::Toolbar::Group  @position="left">
+      <Auk::Toolbar::Item>
+        <Auk::Button
+          @layout="icon-only"
+          @skin="borderless"
+          @icon="chevron-left"
+          @size="large"
+          {{on "click" (perform this.navigateBack)}}
+        />
+      </Auk::Toolbar::Item>
+      <Auk::Toolbar::Item>
+        <div class="auk-overline auk-u-block auk-u-muted">
+          Persberichten
+        </div>
+        <h2 class="auk-toolbar__title">
+          {{this.pressRelease.title}}
+        </h2>
+      </Auk::Toolbar::Item>
+    </Auk::Toolbar::Group >
+     <Auk::Toolbar::Group  @position="right">
+      <Auk::Toolbar::Item>
+        {{#if this.savePressRelease.last.isError}}
+          <Auk::StatusPill @status="error">
+            Opslaan mislukt
+          </Auk::StatusPill>
+        {{else}}
+        <Auk::StatusPill @status="done">
+          {{dayjs-from-now this.pressRelease.modified}} opgeslagen
```

Comment:
Een detail: opgeslagen met hoofdletter (gezien 'Opslaan mislukt' ook met hoofdletter is)